### PR TITLE
Use monkey to start Android applications

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Use the monkey tool to start Android applications on device. This way we do not have to care about the startPackageActivity name and we can start the application in case a custom activity is used.
Remove unused code and variables.

### CODE REVIEW

File: mobile/android/android-application-manager.ts [hunk 1]
```diff
@@ -41,18 +41,10 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
...
-			let startActivityNames = startActivityName ? [startActivityName] : defaultActivityNames;
-
-			_.each(startActivityNames, (activityName: string) => {
-				this.adb.executeShellCommand(["am", "start",
-					"-a", "android.intent.action.MAIN",
-					"-n", `${appIdentifier}/${activityName}`,
-					"-c", "android.intent.category.LAUNCHER"]).wait();
-			});
+			this.adb.executeShellCommand(["monkey",
+				"-p", `${appIdentifier}`,
```
Comment (Mitko-Kerezov) [inline mobile/android/android-application-manager.ts]: I don't think there's a need for the template string here - `appIdentifier` is already of type string

Comment (rosen-vladimirov) [inline mobile/android/android-application-manager.ts]: :smile_cat: thanks for noticing this

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by Mitko-Kerezov** [comment]
👍

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**